solutions/49 - Double Sided Destruction/size-4_speed-13.asm

```diff
@@ -0,0 +1,11 @@
+-- 7 Billion Humans (2087) --
+-- 49: Double Sided Destruction --
+
+-- Author: landfillbaby
+-- Size: 4
+-- Speed: 13
+
+mem1 = nearest shredder
+pickup s
+giveto mem1
+step e
```

Comment:
Wow… this program actually works… what the hell

Reply:
yeah, i was surprised too, i did say in the commit itself thatt it felt like cheating